build.gradle

```diff
@@ -751,20 +753,20 @@ project(':clients') {
   }
 
   task determineCommitId {
-    ext.commitId = "unknown"
```

Comment:
Initially I thought ext.commitId wasn't used since everywhere else were using 'commitId' or determineCommitId.commitId, so I just removed it. Sorry about that. Restored it in the else logic.